_sections/30-projects.md

```diff
@@ -60,6 +60,7 @@ These projects pledge to drop Python 2 support in or before 2020.
 - [![](assets/pymeasure.png)PyMeasure](https://github.com/ralph-group/pymeasure)
 - [![](assets/chaquopy.png)Chaquopy](https://chaquo.com/chaquopy/)
 - [![](assets/angr.png)angr](http://angr.io/)
+- [![](assets/biobuilds.png)angr](https://www.biobuilds.org/)
```

Comment:
You've copied the name `angr` from the line above - that's the text that goes under the logo on the website.

Reply:
Sorry about that. I've pushed a commit to fix it.